### PR TITLE
Update for new HA capabilities

### PR DESCRIPTION
**Pull request by JeffSchering**
This is for [PLT-6080 moving clustering to memberlist #6499](https://github.com/mattermost/platform/pull/6499)

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -215,13 +215,13 @@ Enable Team Creation
...
 
-The default maximum of 50 people, is a the extreme high end of a single team of people. At this point organizations are more often "multiple teams of people" and investments in explicitly defining etiquette, such as `channel organization <https://docs.mattermost.com/help/getting-started/organizing.html>`_ or turning on `policy features <https://docs.mattermost.com/administration/config-settings.html#policy>`_ in Enterprise Edition, are often used to scale the high levels of productivity found in a team of people using Mattermost to multiple teams of people. 
+The default maximum of 50 people, is a the extreme high end of a single team of people. At this point organizations are more often "multiple teams of people" and investments in explicitly defining etiquette, such as `channel organization <https://docs.mattermost.com/help/getting-started/organizing.html>`_ or turning on `policy features <https://docs.mattermost.com/administration/config-settings.html#policy>`_ in Enterprise Edition, are often used to scale the high levels of productivity found in a team of people using Mattermost to multiple teams of people.
```
Comment (lindy65) [inline source/administration/config-settings.rst]: "... is a the extreme high end..." > "... is at the extreme high end..."

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -2307,34 +2307,107 @@ High Availability (Beta)
...
+^^^^^^^^^^^^^^^^^^^^^^^^^
+*Deprecated. Not used in version 4.0 and later*
+
 The address the Mattermost Server will listen on for inter-node communication. When setting up your network you should secure the listen address so that only machines in the cluster have access to that port. This can be done in different ways, for example, using IPsec, security groups, or routing tables.
 
-+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| This feature’s ``config.json`` setting is ``"InterNodeListenAddress": ":8075"`` with string input.                                                                          |
-+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
++-----------------------------------------------------------------------------------------------------+
+| This feature’s ``config.json`` setting is ``"InterNodeListenAddress": ":8075"`` with string innput. |
```
Comment (lindy65) [inline source/administration/config-settings.rst]: Does "innput" have 2 "n's"?

File: source/deployment/cluster-310.rst [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+
+1. Go to **System Console > Configuration** and click **Reload Configuration from Disk** to reload configuration settings for the Mattermost server from ``config.json``.
+2. Go to **System Console > Database** and click **Recycle Database Connections** to take down existing database connections and set up new connections in the multi-database configuration.
+
+While the connection settings are changing, there might be a brief moment when writes to the master database are unsuccessful. The process waits for all existing connections to finish and starts serving new requests with the new connections. End users attempting to send messages while the switch is happening can have an experience similar to losing connection to the Mattermost server.
+
+Transparent Failover
+````````````````````
+
+The database can be configured for high availability and transparent failover use the existing database technologies. We recommend MySQL Clustering, Postgres Clustering, or Amazon Aoura. Database transparent failover is beyond the scope of this documentation.
```
Comment (lindy65) [inline source/deployment/cluster-310.rst]: "...Aoura." > "...Aurora."

File: source/deployment/cluster-310.rst [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+- **Server is down**: If an inter-node communication fails to send a message it makes another attempt in 15 seconds. If the second attempt fails, the server is assumed to be down. An error message is written to the logs and the System Console shows a status of red for that server. The inter-node communication continues to ping the down server in 15 second intervals. When the server comes back up, any new messages are sent to it.
+
+WebSocket Disconnect
+~~~~~~~~~~~~~~~~~~~~
+
+When a client WebSocket receives a disconnect it will automatically attempt to re-establish a connection every three seconds with a backoff. After the connection is established, the client attempts to receive any messages that were sent while it was disconnected.
```
Comment (lindy65) [inline source/deployment/cluster-310.rst]: Is "backoff" one word or two? (or hyphenated?)

Comment (JeffSchering) [inline source/deployment/cluster-310.rst]: In this case, "backoff" is correct.

File: source/administration/config-settings.rst (hunk 1 above)
Comment (JeffSchering) [inline source/administration/config-settings.rst]: ok

File: source/administration/config-settings.rst (hunk 2 above)
Comment (JeffSchering) [inline source/administration/config-settings.rst]: nope

File: source/deployment/cluster-310.rst (hunk 3 above)
Comment (JeffSchering) [inline source/deployment/cluster-310.rst]: thx

**Comment by jasonblais** [comment]
Looks great.

**Pull request merged**